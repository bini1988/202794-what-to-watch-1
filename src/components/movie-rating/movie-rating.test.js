import React from "react";
import renderer from "react-test-renderer";
import MovieRating from "./movie-rating";

it(`MovieRating correctly renders default markup`, () => {
  const tree = renderer.create(
      <MovieRating rating={{
        score: 8.5,
        level: `Good`,
        count: 666,
      }}/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
