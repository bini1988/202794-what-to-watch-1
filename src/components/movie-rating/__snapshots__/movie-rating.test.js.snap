// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieRating correctly renders default markup 1`] = `
<div
  className="movie-rating"
>
  <div
    className="movie-rating__score"
  >
    8.5
  </div>
  <p
    className="movie-rating__meta"
  >
    <span
      className="movie-rating__level"
    >
      Good
    </span>
    <span
      className="movie-rating__count"
    >
      666 ratings
    </span>
  </p>
</div>
`;
